backend/inspirehep/pidstore/converters.py

```diff
@@ -4,13 +4,38 @@
 #
 # inspirehep is free software; you can redistribute it and/or modify it under
 # the terms of the MIT License; see LICENSE file for more details.
+
+from functools import partial
+
 import structlog
-from invenio_records_rest.utils import PIDConverter
+from invenio_records.api import Record
+from invenio_records_rest.utils import PIDConverter, obj_or_import_string
 from werkzeug.routing import PathConverter
 
+from .resolvers import InspireResolver
+
 LOGGER = structlog.getLogger()
 
 
+class InspirePIDConverter(PIDConverter):
+    """Just to use our own Resolver class"""
```

Comment:
unnecessary ;)

Reply:
I think it it, otherwise it's not easy to see why we override PIDConverter. Unless we name it `PidConverterWithCustomResolver` or sth, I'd keep this.